doc/sphinx/Clusters_from_Scratch/active-active.rst

```diff
@@ -4,9 +4,9 @@
 Convert Storage to Active/Active
 --------------------------------
 
-The primary requirement for an Active/Active cluster is that the data
+The primary requirement for an active/active cluster is that the data
 required for your services is available, simultaneously, on both
-machines. Pacemaker makes no requirement on how this is achieved; you
+machines. ``pacemaker`` makes no requirement on how this is achieved; you
```

Comment:
References to the project as a whole would just be the unadorned name "Pacemaker"